common/scripted_effects/wc_sacrifice_scripted_effects.txt

```diff
@@ -257,4 +257,18 @@ PREV_sacrifice_THIS_to_old_god_effect = {
 		}
 		PREV_society_sacrifice_THIS_effect = yes
 	}
+}
+
+sacrifice_all_prisoners_decision_effect = {
+	hidden_effect = {
+		any_courtier = {
+			limit = {
+				THIS_is_prisoner_that_can_be_sacrificed_trigger = yes
+				host = { character = ROOT }
+			}
+			caltulate_PREV_offmap_old_gods_currency_insanity_effect = yes
+			PREV = { add_offmap_old_gods_currency_insanity_after_calculation_effect = yes }
+			PREV_public_sacrifice_THIS_effect = yes
+		}
+	}
 }
```

Comment:
Suggestion: mention a list of courtiers to be sacrificed like it's done in `burn_evil_beings`.